examples/train_tvbr.py

```diff
@@ -0,0 +1,160 @@
+    parser.add_argument("--alpha", nargs="?", type=float, help="ALPHA")
+    return parser.parse_args()
+
+
+class TVBR_train(TrainEngine):
+    """An instance class from the TrainEngine base class."""
+
+    def __init__(self, config):
+        """Initialize configs.
+
+        Args:
+            config (dict): All the parameters for the model
+        """
+        self.config = config
+        super(TVBR_train, self).__init__(self.config)
+        self.load_dataset()
+        self.train_data = self.dataset.sample_triple_time()
+        self.config["alpha_step"] = (1 - self.config["alpha"]) / (
+            self.config["max_epoch"]
+        )
+        self.engine = TVBREngine(self.config)
+
+    def train(self):
+        """Train the TVBR model."""
+        assert hasattr(self, "engine"), "Please specify the exact model engine !"
+        self.monitor = Monitor(
+            log_dir=self.config["run_dir"], delay=1, gpu_id=self.gpu_id
+        )
+        self.engine.data = self.dataset
+        print("Start training... ")
+        epoch_bar = tqdm(range(self.config["max_epoch"]), file=sys.stdout)
+        for epoch in epoch_bar:
+            print(f"Epoch {epoch} starts !")
+            print("-" * 80)
+            if epoch > 0 and self.eval_engine.n_no_update == 0:
+                # previous epoch have already obtained better result
+                self.engine.save_checkpoint(
+                    model_dir=os.path.join(self.config["model_save_dir"], "model.cpk")
+                )
+
+            if self.eval_engine.n_no_update >= MAX_N_UPDATE:
+                print(
+                    "Early stop criterion triggered, no performance update for {:} times".format(
+                        MAX_N_UPDATE
+                    )
+                )
+                break
+            self.engine.train_an_epoch(self.train_data, epoch_id=epoch)
+            self.eval_engine.train_eval(
+                self.dataset.valid[0], self.dataset.test[0], self.engine.model, epoch
+            )
+            # anneal alpha
+            self.engine.model.alpha = min(
+                self.config["alpha"] + math.exp(epoch - self.config["max_epoch"] + 20),
+                1,
+            )
+            """Sets the learning rate to the initial LR decayed by 10 every 10 epochs"""
+            lr = self.config["lr"] * (0.5 ** (epoch // 10))
+            for param_group in self.engine.optimizer.param_groups:
+                param_group["lr"] = lr
+        self.config["run_time"] = self.monitor.stop()
+        return self.eval_engine.best_valid_performance
+
+
+def tune_train(config):
+    """Tune the model.
+
+    Args:
+        config ([type]): [description]
+    """
+    TVBR = TVBR_train(config)
+    best_performance = TVBR.train()
+    tune.track.log(best_ndcg=best_performance)
+    TVBR.test()
+
+
+if __name__ == "__main__":
+    args = parse_args()
+    config = {}
+    update_args(config, args)
```

Comment:
"update_args" is no longer used. Please change to below:
    args = parse_args()
    tvbr = TVBR_train(args)
    tvbr.train()
    tvbr.test()